### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by mike2611**
- Create and run the necessary migration files.
- Table and column match the ERD diagram.
- Foreign keys included.
- All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220114040430_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :postsCounter
```
Comment (Lameck1) [inline db/migrate/20220114040430_create_users.rb:7]: - [x] How about we adhere to ruby/rails naming conventions. `:postsCounter`  should be in snake case i.e. `:posts_counter`. Please make this change.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    t.bigint "post_id", null: false
+    t.index ["post_id"], name: "index_likes_on_post_id"
+    t.index ["user_id"], name: "index_likes_on_user_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.string "title"
+    t.string "text"
+    t.integer "commentsCounter"
+    t.integer "likeCounter"
```
Comment (Lameck1) [inline db/schema.rb]: - [x] Same issue here as above. Use snake case.
  These should be:
  - `comments_counter`
  - `likes_counter`

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    t.string "post_id"
+    t.index ["post_id"], name: "index_posts_on_post_id"
+    t.index ["user_id"], name: "index_posts_on_user_id"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "photo"
+    t.string "bio"
+    t.integer "postsCounter"
```
Comment (Lameck1) [inline db/schema.rb]: - [x] As discussed above. This should be `posts_counter`

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @mike2611, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @mike2611🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**